README.md

```diff
@@ -25,12 +25,10 @@ This library uses semantic versioning and follows Okta's [library version policy
 
 | Okta Spring Boot SDK Version | Compatible Spring Boot Version |
 | ------- | ------------------------- |
-| 0.6 | 1.5.x |
-| 1.0.0, 1.1.0 | 2.1 |
-| 1.2.x | 2.2.0.M3 |
-| 1.3.0, 1.4.0 | 2.2.0.RELEASE |
-| 1.5.0 | 2.4.0 |
- 
+| 1.2.1 | 2.1.2.RELEASE, 2.2.0.M1 |
+| 1.4.0 | 2.2.0.M1, 2.4.0-M1 |
+| 1.5.0 | 2.4.0-M1, 2.5.0-M1 |
+              
```

Comment:
We can leave out the `M` releases as those are milestones and are not supported versions, instead, we could just use ranges, either maven range syntax, `>=`, or maybe just something like `2.2.x - 2.3.x`?

Thoughts?